pico_project.py

```diff
@@ -313,7 +325,8 @@ def __init__(self, parent, initial_config):
     def init_window(self, args):
         self.configure(background=GetBackground())
         self.title("Advanced Configuration")
-        ttk.Label(self, text="Select the advanced options you wish to enable or change. Note that you really should understand the implications of changing these items before using them!").grid(row=0, column=0, columnspan=5)
+        ttk.Label(self, text="Select the advanced options you wish to enable or change. Note that you really should understand the implications of changing these items before using them!").grid(
```

Comment:
If you're adding line-wrapping to stop the line-length going over some limit, it probably makes sense to build this string by joining several sub-strings, or by using the `\` line-continuation?
E.g. https://stackoverflow.com/questions/4172448/is-it-possible-to-break-a-long-line-to-multiple-lines-in-python/4172465